style.css

```diff
@@ -0,0 +1,98 @@
+body{
+  background-color: #F0ffff;
+
```

Comment:
Try to follow a clean code style when it comes to indentation and usage of blank lines. It should be consistent.
Later you can configure the IDE to support on code formatting, but for now try to find a consistent style.